raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
+
+This tutorial assumes raspberry pi to be in the same LAN as your PC with SSH enabled
+These steps have to be run on the raspberry pi 4. The steps to copy files from your `PC` to the `pi` have been explicitly mentioned and have to be run on your `PC`.
+
+`sudo apt update`
+
+`sudo apt full-upgrade`
+
+`sudo apt install git`
+
+Perform a [git install](https://github.com/nvm-sh/nvm#git-install) of the NVM
+
+`git clone https://github.com/nvm-sh/nvm.git .nvm`
+
+`cd ~/.nvm`
+
+`git checkout v0.38.0`
+
+`. ./nvm.sh`
+
+`nvm install --lts`
+
+`npm install --global yarn`
+
+Check the arm system
+
+```
+uname -m
+armv7l
+```
+
+`wget https://gethstore.blob.core.windows.net/builds/geth-linux-arm7-1.10.4-aa637fd3.tar.gz`
```

Comment:
According to https://en.wikipedia.org/wiki/Raspberry_Pi the Pi 4 should have a ARMv8 compatible CPU, so the 64bit version should be used instead.

Reply:
Actually my raspberry pi 4 has a armv7 hence we need to check for it.

Reply:
ok confusion seems to be this the CPU is Armv8 which is a 64bit but the OS is still a armv7 which is 32 bit OS.
The 64bit Raspbian OS  has been launched but its in beta stage.
https://raspberrypi.stackexchange.com/questions/101215/why-raspberry-pi-4b-claims-that-its-processor-is-armv7l-when-in-official-specif